docs/Configuration.md

```diff
@@ -32,6 +32,54 @@ SQLite            | `SQLite`
 
 TBD
 
+## LANGUAGE section
+
+The LANGUAGE section has one key, `lang`.
+
+The value of the `lang` key is a space separated list of
+language tags where each tag must match the regular expression
+`/^[a-z]{2}_[A-Z]{2}$/`.
+
+The two first characters of a language tag are expected to be an
+[ISO 639-1] two-character language code and the two last characters
+are expected to be an [ISO 3166-1 alpha-2] two-character country code.
+A language tag is a locale setting for the available translation
+of messages without ".UTF-8", which is assumed.
+
+Adding a new language to the configuration requires that the
+equivalent MO file is added to Zonemaster-Engine at the correct
+place so that gettext get retrieve it. Removing a language from
+the configuration file just blocks that language from being
+allowed.
+
+English is the Zonemaster default language, but can be blocked
+from being allowed by RPC-API by not including it in the
+configuration.
+
+The default installation and configuration supports the
+following languages.
+
+Language | Language tag value | Locale value used
+---------|--------------------|------------------
+Danish   | da_DK              | da_DK.UTF-8
+English  | en_US              | en_US.UTF-8
+French   | fr_FR              | fr_FR.UTF-8
+Swedish  | sv_SE              | sv_SE.UTF-8
+
+Repeated language tag values are just ignored.
+
+Setting in the default configuration file:
+
+```
+lang = da_DK en_US fr_FR sv_SE
+```
+
+If the lang key is empty or absent, the language tag value
+"en_US" is set by default.
+
+Each locale, including ".UTF-8", set in the configuration file
+must also be installed or activate on the system running the RPCAPI
+daemon for the translation to work correctly.
```

Comment:
This paragraph should probably be removed, and all of its details be integrated in the either part of the split paragraph I requested above, or in the documentation on how to add a new translation.

Reply:
If there was such documentation that would be fine. As it is now I think this is good information here.

Reply:
Fair enough. We should return to this after zonemaster/zonemaster-engine#594 is completed.

Reply:
We should come back to this after zonemaster/zonemaster-enging#594 is completed.